src/common/commonutils/SshUtils.c

```diff
@@ -200,6 +200,78 @@ static int IsSshServerActive(void* log)
     return result;
 }
 
+static int IsSshConfigIncludeSupported(void* log)
+{
+    const char* expectedPrefix = "unknown option -- V OpenSSH_";
+    const char* command = "sshd -V";
+    char* textResult = NULL;
+    size_t textResultLength = 0;
+    size_t textPrefixLength = 0;
+    char* textCursor = NULL;
+    char versionMajorString[2] = {0};
+    char versionMinorString[2] = {0};
+    int versionMajor = 0;
+    int versionMinor = 0;
+    int result = 0;
+    
+    if (false == IsDaemonActive(g_sshServerService, log))
+    {
+        OsConfigLogInfo(log, "IsSshConfigIncludeSupported: the OpenSSH Server service '%s' is not active on this device, assuming Include is not supported", g_sshServerService);
+        return EEXIST;
+    }
+
+    // Extract the two version digits from the SSH server response to unsupported command -V (the OpenSSH way)
+    // For example, version 8.9 from response "unknown option -- V OpenSSH_8.9p1..."
+
+    ExecuteCommand(NULL, command, true, false, 0, 0, &textResult, NULL, NULL);
+
+    if (NULL != textResult)
+    {
+        if (((textPrefixLength = strlen(expectedPrefix)) + 3) < (textResultLength = strlen(textResult))) 
```

Comment:
What is the 3?

Reply:
Two digits + null terminator?

Reply:
Nope. Comment on lines 223-224 explains it. It's the version sequence: "N.N"

Reply:
Also, strlen() does not count the terminator